e2e/cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
```

Comment:
Do we need this file or can we delete it?

Reply:
The rest of these example fixtures can go - but I'd intended to leave them in for test writers to delete as they go.